ios/FluentUI/Tab Bar/TabBarTokens.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `TabBar`.
+open class TabBarTokens: ControlTokens {
+    /// The height of the `TabBar` when in portrait view on a phone
+    open var phonePortraitHeight: CGFloat { 48.0 }
+
+    /// The height of the `TabBar` when in landscape view on a phone
+    open var phoneLandscapeHeight: CGFloat { 40.0 }
+
+    /// The height of the `TabBar` when on a non-phone device
+    open var padHeight: CGFloat { 48.0 }
```

Comment:
even in compact mode of iPad we are always 48?

Reply:
Curious, would you expect it shorter or taller? I think that matches the phone portrait height (and looks like that was the case before tokenizing too); only phone+landscape is shorter.

Reply:
I was operating under the assumption that we were trying not to change any of the look and feel of the controls. If I misunderstood, I'm happy to reference whatever is the correct Figma file and update the value.